labshare-client/src/store/index.js

```diff
@@ -42,17 +39,29 @@ export default new Vuex.Store({
       return state.listData
     },
     authenticated: state => {
-      return state.isAuthenticated
+      return !!state.token
```

Comment:
My understanding is that "double bang"  forces a boolean value, where as `token` could be an empty string or nil, what we want from the function is `true` or `false` rather than true/null/"" etc

Reply:
Makes sense! Good job 👍